### DISCUSSION TITLE
Any inx for building QuSpin dev_0.3.7, Windows 10, VS2022, Python 3.10.5

### DISCUSSION THREAD

**Discussion by nicholasferguson** [QuSpin/QuSpin#553] [Q&A] [answered]
I do not use Anaconda nor the miniAnaconda conda?
With pip,  it does not build.
Anyone can provide inx for a successfull build?  I did try QuSpin.master  That didn't work either.
- even tried command window from VS2022.
Can I ask not to send me inx that someone has not actually implemented themselves? Thanks

**Comment by nicholasferguson** [comment, marked as answer]
Solved this issue.  I recommend you build quspin in windows 10...without anaconda...
First, starting cmd window or powershell from VS2022 does not yield a successful build ?????  I did not revisit this....
1. Start cmd window as admin.  
2. Switch to powershell.  >powershell
3. Run C:\\"Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsamd64_x86.bat"
4	Edit various setup.py  files  ( this is the dirty way.. To add path to BOOST ....  How does conda handle this???
      if sys.platform == "win32":
        extra_compile_args=["-IC:\\temp\ThirdParties\Boost\\boost_1_76_0"]
     Note:  This win32 in x64 is horrible.... But that is a python thing....
6,   run:  python setup.py install --default-compiler-flags --record install_file.txt
To run example0.py	
	   pip install gmpy2
	
	   output:
	   Hermiticity check passed!
	   Symmetry checks passed!
	   Particle conservation check passed!

**Comment by mgbukov** [comment]
Did you try installing from source:

https://weinbe58.github.io/QuSpin/#installation-and-usage

Which platform are you using?
